src/index.html

```diff
@@ -1,17 +1,169 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
+
   <head>
```

Comment:
There should be no blank lines between parent and child blocks, fix it everywhere
```suggestion
<html lang="ru">
  <head>
```